index.html

```diff
@@ -85,5 +85,29 @@
     of code. If it needs to be a bit bigger, it will be.
   </li> 
 </ol> 
+
+<p>
+In addition to these stated goals, we try our best to uphold the following
+values when considering contributions to the project:
+</p>
+
+<ul>
+  <li>
+    Never break configuration files or existing workflows. Breaking changes
+    require a major version bump (v4 → v5).
```

Comment:
Yeah, it's focused on APIs, but in a sense a config (and even more generally, the behavior) is a »U(ser)«PI and I think the concepts still apply fairly well.

> Given that unpublishing a release seems like a confusing move to our users, I wonder what a good position on accidentally incompatible releases would be.

Semantic versioning explicitly mentions this in the FAQ, actually with two different questions depending on whether the breaking change should be kept or not. It's proposal is to immediately release a major version, but leave the broken release as it is (it states that once published, a release is never to be touched again). I think this would make sense and would've made sense for the keyboard problem as well.

That said, I agree that we don't need a strict statement on what versioning concept we use, since we just generally try to avoid breaking changes anyway.